students/slawinski_amadeusz/lesson_06_dicts_tuples_sets_args_kwargs/inventory.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+
+def displayInventory(inventory):
+    print("Inventory:")
+    if inventory is None:
+        print("No items - go hunt some dragons!")
+    else:
+        item_count = 0
+        for k, v in inventory.items():
+            print("{} {}".format(v, k))
+            item_count += v
+        print("Total number of items: " + str(item_count))
```

Comment:
tutaj też mozna wykorzystać `.format()`
` print("Total number of items: {}".format(item_count))`